doc/Makefile

```diff
@@ -7,6 +7,7 @@ SRCDIR           := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME        := $(abspath $(SRCDIR)/..)
 include $(JULIAHOME)/Make.inc
 JULIA_EXECUTABLE := $(call spawn,$(build_bindir)/julia)
+export JULIA_PKGDIR := $(abspath $(SRCDIR)/deps)
```

Comment:
`$(call cygpath_w,$(abspath $(SRCDIR)/deps))`, or set the value of `Base.LOAD_CACHE_PATH[:]` directly in make.jl